JsonLd/ContextBuilder.php

```diff
@@ -119,7 +126,11 @@ public function getContext(ResourceInterface $resource = null)
             }
         }
 
-        return $context;
+        // Dispatch event to allow context customization
+        $event = new Event\ContextBuilderEvent($context);
```

Comment:
The private method "getBaseContext" is also used by "getEntrypointContext", so it can't be moved in the event (or we should type the event: "entrypoint_context" or "resource_context") and dispatch the event in both methods.

And for method "getContext", it depends of the ClassMetadataFactory that should not be injected in the Event IMO.

Instead we could move all this in a Listener, but it sounds a bit overkill. What do you think ?